### DISCUSSION TITLE
Stop on removal RFID Soldering question

### DISCUSSION THREAD

**Discussion by beyondi-ultra** [MiczFlor/RPi-Jukebox-RFID#1437] [Q&A]
Hi, I am not sure whether this is the right place for posting this but I am currently building my firs
t phoniebox  and am looking for some help/guidance regarding soldering.

I have no experience with soldering.
I am looking for someone who can tell me if my RFID reader would support the modification for "stop on removal" (described here: https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Stop-on-removal-with-USB-RFID-Reader ).

If my reader supports this, ideally I am also looking for some recommendations for where I could have this modification done in the UK. Just because I am quite concerned I might break something, don't have any soldering equipment etc.

Any help would be greatly appreciated.

![1](https://user-images.githubusercontent.com/85905617/122072287-5876f180-cdef-11eb-91f3-c0075c30923a.jpg)

![2](https://user-images.githubusercontent.com/85905617/122072325-60369600-cdef-11eb-8828-e4b42a19eb59.jpg)

**Comment by lroeser** [comment]
Hello,

just get a RC522 for 5 bucks, it supports this out of the box (with the latest phoniebox build).

**Reply by beyondi-ultra** [reply]
thanks very much for the reply! I'll look into that
